File: java/a001~a020/a006.java
import java.util.Scanner;

public class a006 {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        Double a, b, c;
        a = sc.nextDouble();
        b = sc.nextDouble();
        c = sc.nextDouble();
        if (b * b - 4 * a * c < 0) {
            System.out.println("No real root");
        }
        if (b * b - 4 * a * c == 0) {
            System.out.printf("Two same roots x=%d", (int) (-b / (2 * a)));
        }
        if (b * b - 4 * a * c > 0) {
            Double temp = (b * b - 4 * a * c);
            System.out.printf("Two different roots x1=%d , x2=%d", (int) ((-b + temp) / (2 * a)),
                    (int) ((-b - temp) / (2 * a)));
        }
        sc.close();
    }
}

File: python/a001~a020/a006.py
a, b, c = map(int, input().split(" "))
if b * b - 4 * a * c < 0:
    print('No real root')
if b * b - 4 * a * c == 0:
    print(f"Two same roots x={int(-b / (2 * a))}")
if b * b - 4 * a * c > 0:
    temp = (b * b - 4 * a * c)**0.5
    print(f"Two different roots x1={int((-b + temp) / (2 * a))} , x2={int((-b - temp) / (2 * a))}")
